cartridges/demo_app_training/cartridge/templates/resources/other.properties

```diff
@@ -1,2 +1,3 @@
 text1.text2=Primeiro
 text3.text4=Segundo {0}
+best.deal=BEST DEAL {0} % OFF
```

Comment:
please move all your changes to app_storefront custom cartridge

Reply:
Should I move all content from my custom cartridge called demo_app_training to app_storefront_custom?

Reply:
yes, please